docs/segments/manage_segments.rst

```diff
@@ -3,4 +3,493 @@
+
+1. In the Campaign builder, click the bottom connector.
+
+  .. image:: images/campaign-builder-connector.png
+    :alt: Screenshot Campaign builder connector.
+
+2. Select **Action**.
+   
+3. In the list of actions, select **Modify Contact’s Segments**.
+
+4. Choose from the list of existing Segments you want to add your Contact to or remove your Contact from.
+
+  .. image:: images/modify-segments.png
+    :alt: Screenshot showing list of existing Segments.
+
+5. Click Save and close.
+
+.. vale off
+
+Form submit action
+==================
+
+.. vale on
+
+**Modify Contact’s Segment** is available as a submit action on Standalone Forms. With a Campaign Form, create a Campaign with a **Modify Contact’s Segment** action.
+
+1. Click **Actions**.
+
+2. From the **Add new submit action** menu, select **Modify Contact’s Segments.**
+
+.. image:: images/form-submit-action.png
+    :alt: Screenshot showing Form submit action.
+
+3. Add a Title for the submit action and add a **Description** (optional).
+
+4. Select the Segments you’d like to add the Contact to or remove the Contact from.
+
+5. Click **Add**.
+
+.. vale off
+
+Points trigger
+==============
+
+.. vale on
+
+Once a Contact has accrued an assigned number of Points, the system can add them to a Segment. This may be a Segment for your most engaged Contacts who become eligible for special offers, or a Segment your sales team reviews to find strong prospects.
+
+.. image:: images/points-trigger.png
+    :alt: Screenshot showing Points trigger.
+
+1. In the **Points** section of the platform, select** Manage Triggers**.
+
+2. Click **+New**.
+
+3. Give your trigger a **Name** and **Description**.
+
+4. Enter the **Minimum number of Points** for adding a Contact to your Segment.
+
+5. Decide if you’d like to add all Contacts with at least that number of Points to the Segment:
+ 
+  * If you only want to add new Contacts who reach the threshold to this Segment, select **No** (default).
+  
+  * To add all existing Contacts with at least a certain number of Points to the Segment, toggle the switch to **Yes**.
+
+6. Click **Events**.
+
+7. Click the menu for **Add an event** and select **Modify Contact’s Segments**.
+
+8.  Add a **Title** for the event and a **Description** (optional).
+
+9.  Select the Segments you’d like to add to or remove the Contact from.
+
+10. Click **Add**.
+
+11. Click **Save & Close**.
+
+This accomplishes the same thing as creating a Dynamic Segment with a filter for Points, the operator ``greater than (or equal to)``, and the minimum number of Points. The difference is if you only want to add Contacts to a Segment who reach the point value after creating this trigger to a Segment, you can.
+
+.. vale off
+
+CSV upload
+===========
+
+.. vale on
+
+If you have a list outside of Mautic, saved as a UTF-8 encoded CSV file, you can upload the list directly into a Mautic Segment.
```

Comment:
Maybe link here to the import section in managing contacts (may not be written yet!) - wonder if this documentation should sit there and we just link to it? What do you think? 

There's stuff like using the import in the background cron which are not relevant to add here (we just want to focus on the bit about how you get them into a segment), so I think it makes sense to point the user to the more in-depth docs.

Reply:
yeah, this section is in the contacts documentation which has not been written yet